src/molecule_plugins/podman/playbooks/create.yml

```diff
@@ -149,6 +149,8 @@
         hostname: "{{ item.hostname | default(omit) }}"
         image: "{{ item.pre_build_image | default(false) | ternary('', 'molecule_local/') }}{{ item.image }}"
         ip: "{{ item.ip | default(omit) }}"
+        labels:
```

Comment:
Why using ``labels`` alias instead of ``label`` ?

Reply:
no particular reason. amended to use canonical `label` in 68ac3a4.